default/python/universe_generation/galaxy.py

```diff
@@ -1,35 +1,297 @@
 import sys
-from random import random, uniform, randint, gauss, choice
-from math import pi, sin, cos, sqrt
+from random import random, uniform, randint, gauss
+from math import pi, sin, cos, acos, sqrt, ceil, floor
+from collections import defaultdict
 
 import freeorion as fo
 import util
 import universe_tables
 
 
-class AdjacencyGrid:
+class AdjacencyGrid(object):
     def __init__(self, universe_width):
+        assert universe_tables.MIN_SYSTEM_SEPARATION < universe_tables.MAX_STARLANE_LENGTH
```

Comment:
In the description of your PR you say that this additional requirement to be
`MAX_STARLANE_LENGTH >= 2 * MIN_SYSTEM_SEPARATION`
So either there is a `*2` missing here after `MIN_SYSTEM_SEPARATION`, or the description is wrong, or I'm missing something obvious...?